Cargo.toml

```diff
@@ -18,4 +18,5 @@ rand_mt = "4.2"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 tracing = "0.1"
-reliquary-proc-macro = { path = "reliquary-proc-macro" }
\ No newline at end of file
+reliquary-proc-macro = { path = "reliquary-proc-macro" }
+anyhow = "1.0.86"
```

Comment:
`thiserror` is better for libraries. `anyhow` is a quick and dirty solution for applications. It's not great for libraries because it gets exposed to the caller, and because you can't check the actual error condition.